web/src/typeahead_helper.ts

```diff
@@ -163,6 +163,7 @@ export let render_person = (person: UserPillData | UserOrMentionPillData): strin
         ),
         pronouns,
         secondary: person.user.delivery_email,
+        show_comma: person.user.delivery_email !== null,
```

Comment:
Hmm, looking at this more, I think `has_secondary` should have been `true` with the previous code. I think the other bug here `has_secondary = args.secondary !== undefined` should be `args.secondary !== undefined && args.secondary !== null`. (We probably don't we want `0` to be treated differently).